CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.7.0 [2022-08-17]
+
+_Bug fixes_
+
+- Fixed the `net_certificate` table to return an empty row instead of an error if the domain did not have an SSL certificate. ([#42](https://github.com/turbot/steampipe-plugin-net/pull/42))
```

Comment:
```suggestion
- Fixed the `net_certificate` table to return an empty row instead of an error if the domain does not have an SSL certificate. ([#42](https://github.com/turbot/steampipe-plugin-net/pull/42))
```